### ISSUE TITLE
Not compatible with SSMS 18.9.2

### ISSUE THREAD

**Issue by LurchUSA** [ErikEJ/SqlCeToolbox#912]
When I try to run the vsix installer, I get a warning message:
![image](https://user-images.githubusercontent.com/21182290/133807697-01cc286d-c1c0-4943-9779-df9fabeb2f51.png)

Is your tool out of date?

**Comment by ErikEJ** [comment]
@LurchUSA Maybe. Where did you get the vsix from?

**Comment by ErikEJ** [comment]
Ping?

**Comment by ErikEJ** [comment]
Closing, feel free to re-open if you can provide the requested info.

**Comment by Benchwork** [comment]
I attempted to download and install this from [Here marketplace.visualstudio.com](https://marketplace.visualstudio.com/items?itemName=ErikEJ.SQLServerCompactSQLiteToolboxforSSMS)
I got the same error, below is the error log 

9/3/2022 1:26:28 PM - Microsoft VSIX Installer
9/3/2022 1:26:28 PM - -------------------------------------------
9/3/2022 1:26:28 PM - vsixinstaller.exe version:
9/3/2022 1:26:28 PM - 17.3.2092
9/3/2022 1:26:28 PM - -------------------------------------------
9/3/2022 1:26:28 PM - Command line parameters:
9/3/2022 1:26:28 PM - C:\Program Files (x86)\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\VSIXInstaller.exe,C:\Users\uchrp001\Downloads\SSMSToolbox (1).vsix
9/3/2022 1:26:28 PM - -------------------------------------------
9/3/2022 1:26:28 PM - Microsoft VSIX Installer
9/3/2022 1:26:28 PM - -------------------------------------------
9/3/2022 1:26:28 PM - Initializing Install...
9/3/2022 1:26:28 PM - Extension Details...
9/3/2022 1:26:28 PM - 	Identifier         : d6c77c32-fe4b-4f6d-ad5d-f7b755212760
9/3/2022 1:26:28 PM - 	Name               : SQL Server Compact/SQLite Toolbox for SSMS
9/3/2022 1:26:28 PM - 	Author             : ErikEJ
9/3/2022 1:26:28 PM - 	Version            : 4.7.632
9/3/2022 1:26:28 PM - 	Description        : SQL Server Compact/SQLite Toolbox add-in for SQL Server Management Studio. This add-in adds several features to help your embedded database development efforts: Scripting of tables and data, import from SQL Server and CSV files and much, much more.
9/3/2022 1:26:28 PM - 	Locale             : en-US
9/3/2022 1:26:28 PM - 	MoreInfoURL        : https://github.com/ErikEJ/SqlCeToolbox
9/3/2022 1:26:28 PM - 	InstalledByMSI     : False
9/3/2022 1:26:28 PM - 	SupportedFrameworkVersionRange : [4.5,)
9/3/2022 1:26:28 PM - 
9/3/2022 1:26:28 PM - 	SignatureState     : Unsigned
9/3/2022 1:26:28 PM - 	Supported Products : 
9/3/2022 1:26:28 PM - 		ssms
9/3/2022 1:26:28 PM - 			Version : [14.0,15.0)
9/3/2022 1:26:28 PM - 			ProductArchitecture : x86
9/3/2022 1:26:28 PM - 		Microsoft.VisualStudio.IntegratedShell
9/3/2022 1:26:28 PM - 			Version : [14.0,15.0)
9/3/2022 1:26:28 PM - 			ProductArchitecture : x86
9/3/2022 1:26:28 PM - 
9/3/2022 1:26:28 PM - 	References         : 
9/3/2022 1:26:28 PM - 		-------------------------------------------------------
9/3/2022 1:26:28 PM - 		Identifier   : Microsoft.VisualStudio.MPF.14.0
9/3/2022 1:26:28 PM - 		Name         : Visual Studio MPF 14.0
9/3/2022 1:26:28 PM - 		Version      : [14.0]
9/3/2022 1:26:28 PM - 		MoreInfoURL  : 
9/3/2022 1:26:28 PM - 		Nested       : No
9/3/2022 1:26:28 PM - 
9/3/2022 1:26:28 PM - 	Prerequisites      : 
9/3/2022 1:26:28 PM - 		-------------------------------------------------------
9/3/2022 1:26:28 PM - 		Identifier   : Microsoft.VisualStudio.Component.CoreEditor
9/3/2022 1:26:28 PM - 		Name         : Visual Studio core editor
9/3/2022 1:26:28 PM - 		Version      : [14.0,16.0)
9/3/2022 1:26:28 PM - 
9/3/2022 1:26:28 PM - Signature Details...
9/3/2022 1:26:28 PM - 	Extension is not signed.
9/3/2022 1:26:28 PM - 
9/3/2022 1:26:28 PM - Searching for applicable products...
9/3/2022 1:26:28 PM - Found installed product - Global Location
9/3/2022 1:26:28 PM - Found installed product - Visual Studio Community 2022
9/3/2022 1:26:28 PM - VSIXInstaller.NoApplicableSKUsException: This extension is not installable on any currently installed products.
   at VSIXInstaller.ExtensionService.GetInstallableDataImpl(IInstallableExtension extension, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionService.GetInstallableData(String vsixPath, String extensionPackParentName, Boolean isRepairSupported, IStateData stateData, IEnumerable`1& skuData)
   at VSIXInstaller.ExtensionPackService.IsExtensionPack(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.ExtensionPackService.ExpandExtensionPackToInstall(IStateData stateData, Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize(Boolean isRepairSupported)
   at VSIXInstaller.App.Initialize()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
